artox-lab/clarc-bundle/0.1/config/packages/artox_lab_clarc.yaml

```diff
@@ -0,0 +1,4 @@
+artox_lab_clarc:
+    api:
+        serializer:
+            class: \ArtoxLab\Bundle\ClarcBundle\Core\Interfaces\UI\API\Transformers\Serializers\NullObjectArraySerializer
```

Comment:
Wow, that is a long namespace =)

Reply:
I think it ok, because 4 depths (ArtoxLab\Bundle\ClarcBundle\Core) is only prefix to bundle abstracts